src/AtmosphereRenderer.cpp

```diff
@@ -34,24 +35,24 @@
 
 namespace csp::atmospheres {
 
-AtmosphereRenderer::AtmosphereRenderer(std::shared_ptr<Plugin::Properties> const& pProperties)
-    : mProperties(pProperties) {
+AtmosphereRenderer::AtmosphereRenderer(std::shared_ptr<Plugin::Settings> const& pSettings)
+    : mPluginSettings(pSettings) {
```

Comment:
```suggestion
AtmosphereRenderer::AtmosphereRenderer(std::shared_ptr<Plugin::Settings> pSettings)
    : mPluginSettings(std::move(pSettings)) {
```

Reply:
You're right but in order to avoid too many conflicts with the `feature/clang-tidy` I will do this after it got merged.